src/index.html

```diff
@@ -3,33 +3,104 @@
     </head>
     <body>
-        <main class="container">
-            <img src="images/favicon-500.png" width="300" height="300" alt="Stockpile Logo">
-            <h1>Stockpile</h1>
-            <h2>Manage your stuff</h2>
-            Contact <a mailto="info@stockpileapp.co">info@stockpileapp.co</a> for support.
+        <nav>
+            <div class="container">
+                <h1>
+                    <a href="/" class="stockpile-logo"></a>
+                </h1>
+            </div>
+        </nav>
+        <header class="container">
+            <div class="row around-xs">
+                <div class="col-xs-3">
+                    <div class="tipi"></div>
+                </div>
+                <div class="col-xs-12 center-xs col-md-6 start-md">
+                    <h2>Manage your stuff.</h2>
+                    <p>
+                        Stockpile is the simplest way to manage your inventory,
+                        no matter what you have on your shelves.
+                    </p>
+                    <a href="/subscribe">
+                        <button class="button-primary">Get Stockpile</button>
+                    </a>
+                </div>
+            </div>
+        </header>
+        <main>
+            <section class="row center-xs middle-md">
+                <div class="col-md-offset-2 col-xs-6 col-md-3">
+                    <p>
+                        With Stockpile, you can add any item with a barcode on
+                        it to your inventory.
+                    </p>
+                    <p>
+                        Each item has a brand, model, and
+                        category.
+                    </p>
+                    <p>
+                        Use these fields to filter your inventory to
+                        find exactly the item you need, when you need it.
+                    </p>
+                </div>
+                <div class="col-xs-6 col-md-3">
+                    <div class="stockpile-inventory"></div>
+                </div>
+            </section>
+            <section class="row center-xs middle-md">
+                <div class="col-md-offset-2 col-xs-6 col-md-3">
+                    <div class="stockpile-inventory"></div>
+                </div>
+                <div class="col-xs-6 col-md-3">
+                    <p>
+                        To rent an item, all you have to do is scan its barcode
+                        and select a return date.
+                    </p>
+                    <p>
+                        To return items, simply scan them again.
+                    </p>
+                </div>
+            </section>
+            <section class="row center-xs middle-md">
+                <div class="col-md-offset-2 col-xs-6 col-md-3">
+                    <p>
+                        Create kits of items that go together so you don't forget
+                        anything when you rent items.
```

Comment:
I think reading through the different sections, the word 'item' is repeated a lot. Is there a way we can rephrase some of them to make it flow a little better?